.vib/phpmyadmin/cypress/cypress/integration/phpmyadmin_spec.js

```diff
@@ -1,75 +1,75 @@
+  cy.contains('#login_form', 'Log in');
+});
 
 it('allows creating a database and a table', () => {
-    cy.login();
-    cy.visit('index.php?route=/server/databases');
-    cy.fixture('testdata').then((td) => {
-        cy.get('#text_create_db').type(`${td.databaseName}.${random}`);
-        cy.get('#buttonGo').click();
-        cy.get('.lock-page [type="text"]').type(`${td.tableName}.${random}`);
-        cy.get('.lock-page [type="number"]').clear().type(td.columnNumber);
-        cy.contains('[type="submit"]', 'Go').click();
-        cy.get('#field_0_1').type(`${td.columnName}.${random}`);
-        cy.get('.btn-primary').click();
-        cy.visit('index.php');
-        cy.contains('a', `${td.databaseName}.${random}`).scrollIntoView().click({
-            force: true
-        });
-        cy.contains('#pma_navigation_tree_content', `${td.tableName}.${random}`).click({
-            force: true
-        })
-        cy.contains('.table-responsive-md', `${td.columnName}.${random}`).should('be.visible');
+  cy.login();
+  cy.visit('index.php?route=/server/databases');
+  cy.fixture('testdata').then((td) => {
+    cy.get('#text_create_db').type(`${td.databaseName}.${random}`);
+    cy.get('#buttonGo').click();
+    cy.get('.lock-page [type="text"]').type(`${td.tableName}.${random}`);
+    cy.get('.lock-page [type="number"]').clear().type(td.columnNumber);
+    cy.contains('[type="submit"]', 'Go').click();
+    cy.get('#field_0_1').type(`${td.columnName}.${random}`);
+    cy.get('.btn-primary').click();
+    cy.visit('index.php');
+    cy.contains('a', `${td.databaseName}.${random}`).scrollIntoView().click({
+      force: true,
+    });
+    cy.contains(
+      '#pma_navigation_tree_content',
+      `${td.tableName}.${random}`
+    ).click({
+      force: true,
     });
-})
+    cy.contains('.table-responsive-md', `${td.columnName}.${random}`);
+  });
+});
 
 it('allows importing a table and executing a query', () => {
-    cy.login();
-    cy.visit('index.php?route=/database/structure&server=1&db=mysql')
-    cy.contains('Import').click();
-    cy.get('#input_import_file').selectFile('cypress/fixtures/testdata.sql', {
-        force: true
-    });
-    cy.get('#buttonGo').click();
-    cy.contains('No database selected', '[role="alert"]').should('not.exist');
-    cy.contains('Import has been successfully finished');
-    cy.fixture('testdata').then((td) => {
-        cy.contains('[title="Browse"]', td.importedDatabaseName).should('be.visible').click();
-    })
-    cy.contains('#topmenu', 'SQL');
-    cy.visit('/index.php?route=/table/sql&db=mysql&table=authors');
-    cy.get('#button_submit_query').scrollIntoView().click();
-    cy.get('.result_query').contains('Showing rows');
-})
+  cy.login();
+  cy.visit('index.php?route=/database/structure&server=1&db=mysql');
+  cy.contains('Import').click();
+  cy.get('#input_import_file').selectFile('cypress/fixtures/testdata.sql', {
+    force: true,
+  });
+  cy.get('#buttonGo').click();
```

Comment:
Looking at this code, could we avoid using CSS to identify some of the elements? I'm seeing a lot of selectors used and I wonder if this is because of the app design or something we missed when the tests were added.

Reply:
In this case, it's the app design. I've had this in mind when reviewing this code and I have tried my best to apply changes whenever it was feasible. 
However, this application has an old UI with many buttons/fields/equally named elements on one page.
So if I would use something like `cy.contains('button',Save')` it would yield multiple elements. 
This app also had a lot of flakiness issues so CSS seems like a safer choice maintenance-wise.